### ISSUE TITLE
Output examples in help

### ISSUE THREAD

**Issue by egoist** [cacjs/cac#5]
```js
// global examples
cli.example(`{bin} init project-name --force`)

// command examples
const command = cli.command('build')
command.example('{bin} build src/index.js')
```

**Comment by hugomd** [comment]
@egoist I'd be interested in helping out with this 👍

**Comment by egoist** [comment]
@hugomd that would be nice.

**some more notes:**

**==> *replace bin*:**

```js
cli.example('{bin} init project-name --force')
```

We replace the `{bin}` here with: https://github.com/cacjs/cac/blob/325be30a7f377fd0127b024614b4ae6c37ec0cfe/src/Cac.js#L18

**==> *add description for example*:**

```js
cli.example('{bin} init --force', 'Init a project even if it already exists')
```

**Comment by egoist** [comment]
Maybe this can even be a plugin:

<img width="668" alt="2017-10-25 11 16 18" src="https://user-images.githubusercontent.com/8784712/32007026-f61447da-b9da-11e7-9d88-d3762ff60198.png">

**Comment by egoist** [comment]
Landed in `v4.3.0`

https://github.com/cacjs/cac/blob/master/examples/show-examples.js
